test/Polly.Core.Tests/Hedging/HedgingResilienceStrategyTests.cs

```diff
@@ -634,10 +634,10 @@ public async Task ExecuteAsync_EnsureBackgroundWorkInSuccessfulCallNotCancelled(
         // assert
         _timeProvider.Advance(TimeSpan.FromDays(2));
 
-        await Assert.ThrowsAsync<TaskCanceledException>(() => backgroundTasks[0]);
+        await backgroundTasks[0].Invoking(async t => await t).Should().ThrowAsync<TaskCanceledException>();
```

Comment:
IMHO this specific xunit assertion is superior to the FluentAssertions style.

Having to async await `t` is just visual noise.

Could you revert this one please?

Reply:
I do agree that FluentAssertions style is really verbose compared to Xunit's assertions.

I've replaced this `Assert.ThrowsAsync` statement because inside this test file the majority of the assertions were done via FluentAssertions.

I'm fine with both style for checking the thrown exception but I would prefer consistent usage (at least inside the test class). 

Shall I rewrite all `.Should().ThrowAsync` to `Assert.ThrowsAsync` inside `HedgingResilienceStrategyTests`?

Reply:
Just change this one for now so we can get the merge done, then the others can be refactored separately.